pkg/scheduler/apis/config/types.go

```diff
@@ -220,9 +220,20 @@ type PluginConfig struct {
 	Args runtime.Unknown
 }
 
-///////////////////////////////////////////////////////////////////////////////
-// NOTE: The following methods are intentionally left out of the staging mirror.
-///////////////////////////////////////////////////////////////////////////////
+/*
+ * NOTE: The following variables and methods are intentionally left out of the staging mirror.
+ */
+const (
+	// DefaultPercentageOfNodesToScore defines the percentage of nodes of all nodes
+	// that once found feasible, the scheduler stops looking for more nodes.
+	DefaultPercentageOfNodesToScore = 50
+
+	// MaxCustomPriorityScore is the max score UtilizationShapePoint expects.
+	MaxCustomPriorityScore int64 = 10
```

Comment:
We wanted to increase the range for internal priorities to [0, 100], however, we can't force that on custom priorities since that breaks compatibility, so we defined a separate max priority for them. Internally, we scale it to the proper range when aggregating the scores: https://github.com/kubernetes/kubernetes/blob/c08161a7812c0e850c2614699a6d62eae459810b/pkg/scheduler/algorithm_factory.go#L427

Reply:
and this is here because in some sense it is part of the API if that makes sense.

Reply:
> Internally, we scale it to the proper range when aggregating the scores

is the weight specified by the custom priority policy scaled as well?

Reply:
no, weights are supposed to be relative not absolute, right?

Reply:
> weights are supposed to be relative not absolute, right?

I'm not familiar enough to know. Are there tests in place that ensure a custom priority combined with internal priorities continues to have the same effect/weight after this change as before?

Reply:
Yes there are tests for that.